pyproject.toml

```diff
@@ -1,20 +1,24 @@
-[tool.poetry]
+[project]
 name = "fnv-hash-fast"
 version = "1.5.0"
-description = "A fast version of fnv1a"
-authors = ["J. Nick Koston <nick@koston.org>"]
 license = "MIT"
+description = "A fast version of fnv1a"
 readme = "README.md"
-repository = "https://github.com/bluetooth-devices/fnv-hash-fast"
+authors = [{ name = "J. Nick Koston", email = "nick@koston.org" }]
+requires-python = ">=3.10"
+
+[project.urls]
+"Repository" = "https://github.com/bluetooth-devices/fnv-hash-fast"
+"Bug Tracker" = "https://github.com/bluetooth-devices/fnv-hash-fast/issues"
+"Changelog" = "https://github.com/bluetooth-devices/fnv-hash-fast/blob/main/CHANGELOG.md"
+
+[tool.poetry]
 classifiers = [
     "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
     "Natural Language :: English",
     "Operating System :: OS Independent",
     "Topic :: Software Development :: Libraries",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python :: 3.13",
```

Comment:
Note, the classifiers here only list 3.11+. However `tool.poetry.dependencies.python` is defined as `^3.10`, so the classifier for `3.10` was added automatically anyway.